### PR TITLE
cmd/snap-seccomp/syscalls: update seccomp system call list

### PR DESCRIPTION
**Pull request by Meulengracht**
Update to this commit: https://github.com/seccomp/libseccomp/commit/aa168d49243b95f63b9825a87351a1eb323dc792

Fix for google:ubuntu-18.04-64:tests/main/snap-seccomp-syscalls

### CODE REVIEW

File: cmd/snap-seccomp/syscalls/syscalls.go [hunk 1]
```diff
@@ -36,6 +36,8 @@ var SeccompSyscalls = []string{
 	"arch_prctl",
 	"arm_fadvise64_64",
 	"arm_sync_file_range",
+	"atomic_barrier",
```
Comment (mvo5) [inline cmd/snap-seccomp/syscalls/syscalls.go:39]: The comment above this says: ```
Generated using arch-syscall-dump test tool from libseccomp tree, git
// revision bf747eb21e428c2b3ead6ebcca27951b681963a0.
var SeccompSyscalls = []string{
```
do we need to update the git revision of this comment?

**Review by mvo5 — COMMENTED** [review]
Thanks for looking into this!

File: cmd/snap-seccomp/syscalls/syscalls.go (hunk 1 above)
Comment (Meulengracht) [reply cmd/snap-seccomp/syscalls/syscalls.go:39]: yea probably good idea

**Review by mvo5 — APPROVED** [review]
Thank you

**Pull request merged**